Tests/Foundation/Tests/TestURLSession.swift

```diff
@@ -1922,6 +2007,13 @@ class TestURLSession: LoopbackServerTest {
             /* ⚠️ */      testExpectedToFail(test_noDoubleCallbackWhenCancellingAndProtocolFailsFast, "This test crashes nondeterministically: https://bugs.swift.org/browse/SR-11310")),
             /* ⚠️ */ ("test_cancelledTasksCannotBeResumed", testExpectedToFail(test_cancelledTasksCannotBeResumed, "Breaks on Ubuntu 18.04")),
         ]
+        if #available(macOS 12.0, *) {
```

Comment:
I think this was macOS 10.15 in the earlier #available() checks

Reply:
WebSocket support requires 10.15.
XCTestCase.asyncTest requires 12.0